src/ops.jl

```diff
@@ -189,7 +184,7 @@ trace(A::FixedMatrix{2,2}) = A[1,1] + A[2,2]
 trace(A::FixedMatrix{3,3}) = A[1,1] + A[2,2] + A[3,3]
 trace(A::FixedMatrix{4,4}) = A[1,1] + A[2,2] + A[3,3] + A[4,4]
 
-\{m,n,T}(mat::Mat{m,n,T}, v::Vec{n, T}) = inv(mat)*v
+\{m,n,T1,T2}(mat::Mat{m,n,T1}, v::Vec{n,T2}) = inv(mat)*v
```

Comment:
Yes, I noticed this too, it looks dodgy from a stability point of view too, though I assume not too bad for very small matrices (?)